cross/ffmpeg/Makefile

```diff
@@ -104,6 +104,8 @@ endif
 DEPENDS += cross/libzmq
 CONFIGURE_ARGS += --enable-libzmq
 
+DEPENDS += cross/zvbi
+CONFIGURE_ARGS += --enable-libzvbi
```

Comment:
Does it build on all archs?  I see a few failures that may lead into adding `ifeq` to allow building only for specific arches (or discard other, as you see fit).

Reply:
Hard to say if it builds on all platform. I can start a all-supported build on my docker image. The github build failed , but due to it recompiled  chromaprint that depends on ffmpeg.

Reply:
Lets first hit a fully successful build on github-action.
Current master do build successfully (see https://github.com/SynoCommunity/spksrc/pull/4866).